lib/time/doc.go

```diff
@@ -0,0 +1,56 @@
+/*Package time defines time primitives for starlark, based heavily on the time
+package from the go standard library.
+
+  outline: time
+    time defines time primitives for starlark
+    path: time
+    functions:
+      duration(string) duration
+        parse a duration
+      location(string) location
+        parse a location
+      time(string, format=..., location=...) time
+        parse a time
+      fromtimestamptime(int) time
+        parse a Unix timestamp
+      now() time
+        implementations would be able to make this a constant
+      zero() time
+        a constant
+
+    types:
+      duration
+        fields:
+          hours float
+          minutes float
+          nanoseconds int
+          seconds float
+        operators:
+          duration - time = duration
```

Comment:
IMO this is not really a sensible operation.

Reply:
I agree.